src/citrine/informatics/design_candidate.py

```diff
@@ -124,6 +126,14 @@ def __init__(self, *, smiles: str):
 class DesignMaterial(Serializable["DesignMaterial"]):
     """Description of the material that was designed, as a set of DesignVariables."""
 
+    material_id = properties.UUID('identifiers.id')
```

Comment:
The backend populates these fields already. They were just being ignored before.